CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**4.1.0 - 05/14/25**
```

Comment:
This is a new feature and dependent upon the vivarium update but I would also be fine with a patch release